zerver/openapi/zulip.yaml

```diff
@@ -7311,9 +7311,13 @@ paths:
                       ignored_parameters_unsupported: {}
                       avatar_url:
                         type: string
+                        nullable: true
                         description: |
                           URL for the user's avatar.
 
+                          **Changes**: In Zulip 7.0 (feature level 177)
+                          this will be `null` when client can compute the URL.
+
```

Comment:
How does this PR changes the response of the `users/me` endpoint?

Reply:
We are always passing `client_gravatar=False` to `get_raw_user_data` from `get_profile_backend`.